### PR TITLE
Rename FlowUI classes and properties

### PR DESCRIPTION
**Pull request by glebfox**
Fixes: jmix-framework/jmix#1830

### CODE REVIEW

File: jmix-security/security-flowui/src/main/java/io/jmix/securityflowui/model/RoleModelConverter.java [hunk 1]
```diff
@@ -30,7 +30,7 @@
  * into non-persistent {@link ResourceRoleModel}, {@link RowLevelRoleModel} entities which may be
  * displayed in UI
  */
-@Component("sec_FlowuiRoleModelConverter")
+@Component("sec_UiRoleModelConverter")
```
Comment (knstvk) [inline jmix-security/security-flowui/src/main/java/io/jmix/securityflowui/model/RoleModelConverter.java:33]: `@Component("sec_RoleModelConverter")`

**Review by knstvk — CHANGES_REQUESTED** [review]

File: jmix-flowui/flowui/src/main/java/io/jmix/flowui/FlowuiConfiguration.java [hunk 2]
```diff
@@ -44,7 +44,7 @@
 @ComponentScan
 @ConfigurationPropertiesScan
 @JmixModule(dependsOn = CoreConfiguration.class)
-@PropertySource(name = "io.jmix.flowui", value = "classpath:/io/jmix/flowui/module.properties")
+@PropertySource(name = "io.jmix.ui", value = "classpath:/io/jmix/flowui/module.properties")
```
[the remark this answers is not in the document]
Comment (glebfox) [reply jmix-flowui/flowui/src/main/java/io/jmix/flowui/FlowuiConfiguration.java:47]: miss fix

File: jmix-flowui/flowui-data/src/main/java/io/jmix/flowuidata/accesscontext/UiGenericFilterModifyGlobalConfigurationContext.java [hunk 3]
```diff
@@ -18,11 +18,11 @@
 
 import io.jmix.core.accesscontext.SpecificOperationAccessContext;
 
-public class FlowuiGenericFilterModifyGlobalConfigurationContext extends SpecificOperationAccessContext {
+public class UiGenericFilterModifyGlobalConfigurationContext extends SpecificOperationAccessContext {
 
-    public static final String NAME = "flowui.genericfilter.modifyGlobalConfiguration";
+    public static final String NAME = "ui.genericfilter.modifyGlobalConfiguration";
```
Comment (glebfox) [inline jmix-flowui/flowui-data/src/main/java/io/jmix/flowuidata/accesscontext/UiGenericFilterModifyGlobalConfigurationContext.java:23]: Should we rename it to `ui.filter.modifyGlobalConfiguration` in respect with `ui.filter.modifyJpqlCondition` and `ui.filter.modifyConfiguration`?

**Comment by knstvk** [comment]
👍

**Pull request closed without merging**